assets/js/blocks/attribute-filter/block.js

```diff
@@ -392,9 +397,20 @@ const AttributeFilterBlock = ( {
 					getQueryArgs( window.location.href )
 				);
 
+				const parsedTaxonomy = parseTaxonomyToGenerateURL(
+					attributeObject?.taxonomy
+				);
+
 				const url = currentQueryArgKeys.reduce(
 					( currentUrl, queryArg ) =>
-						removeQueryArgs( currentUrl, queryArg ),
```

Comment:
I updated this logic because with the previous implementation, ALL query parameters were removed, instead only the query parameters related to the attribute filter have to be removed.